Pipfile

```diff
@@ -15,7 +15,7 @@ pandas = "*"
 requests = "*"
 apache-airflow = "*"
 cryptography = "*"
-socrata-py = "*"
+socrata-py = { git = "https://github.com/sbuss/socrata-py.git" }
```

Comment:
Hmm how does the infinite recursion bug manifest? Is it a blocker to us getting the dashboard going? For a project where we are going to disappear in a week I'm hesitant to point at a fork which may not get updated any time soon?

Reply:
It's a blocker for uploading the schemaless file to datasf. it crashes :\

Reply:
Ahh bummer, ok can we file a high priority GitHub issue to move this back ASAP once your fix is merged in Socrata and also let Mike/Jason know that we have to do this the next time we talk to them? Approving this for merge in meantime

Reply:
Created #180